public/app/features/dimensions/editors/TextDimensionEditor.tsx

```diff
@@ -81,12 +88,17 @@ export const TextDimensionEditor: FC<StandardEditorProps<TextDimensionConfig, Te
       {mode === TextDimensionMode.Fixed && (
         <InlineFieldRow>
           <InlineField label={'Value'} labelWidth={labelWidth} grow={true}>
-            <StringValueEditor
-              context={context}
-              value={value?.fixed}
-              onChange={onFixedChange}
-              item={dummyStringSettings}
-            />
+            <>
+              <StringValueEditor
```

Comment:
maybe the X can be internal to this component?  maybe add a setting for `isClearable`?